code/game/machinery/constructable_frame.dm

```diff
@@ -445,6 +445,45 @@ to destroy them and players will be able to make replacements.
 	origin_tech = "programming=1"
 	req_components = list(
 							/obj/item/weapon/stock_parts/matter_bin = 1)
+	var/list/fridge_names_paths = list(
+							"\improper SmartFridge" = /obj/machinery/smartfridge,
+							"\improper MegaSeed Servitor" = /obj/machinery/smartfridge/seeds,
+							"\improper Refrigerated Medicine Storage" = /obj/machinery/smartfridge/medbay,
+							"\improper Slime Extract Storage" = /obj/machinery/smartfridge/secure/extract,
+							"\improper Secure Refrigerated Medicine Storage" = /obj/machinery/smartfridge/secure/medbay,
+							"\improper Smart Chemical Storage" = /obj/machinery/smartfridge/secure/chemistry,
+							"smart virus storage" = /obj/machinery/smartfridge/secure/chemistry/virology,
+							"\improper Drink Showcase" = /obj/machinery/smartfridge/drinks
+	)
+
+
+/obj/item/weapon/circuitboard/smartfridge/New()

```

Comment:
feels like i should do something with this and [this](https://github.com/ParadiseSS13/Paradise/blob/master/code/game/machinery/constructable_frame.dm#L914)